code/src/components/NoData.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components'
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+`
+
+const Text = styled.p`
+  font-size: 25px;
+  text-align: center;
+  margin: 60px 40px 0px 40px;
+  color: #032640;
+`
+
+const NoData = () => {
+  const items = useSelector((state) => state.todos.items)
+
+  return (
+    <>
+    {!items.length && (
```

Comment:
it should be checked in the parent component if it is needed because it would save resources used to mount this component😭